tests/e2e/e2e_test.go

```diff
@@ -276,6 +276,32 @@ var _ = Describe("odoe2e", func() {
 			timeoutValue := getPreferenceValue("Timeout")
 			Expect(timeoutValue).To(BeEmpty())
 		})
+
+		It("should set env variables", func() {
+			runCmdShouldPass("odo config set --env PORT=4000")
+			configValue := runCmdShouldPass("odo config view")
+			Expect(configValue).To(ContainSubstring("PORT"))
+			runCmdShouldPass("odo config set --env SECRET_KEY=R2lyaXNoIFJhbW5hbmkgaXMgdGhlIGJlc3Q=")
```

Comment:
$ echo R2lyaXNoIFJhbW5hbmkgaXMgdGhlIGJlc3Q= | base64 --decode
Girish Ramnani is the best